src/main/java/com/ibm/watson/modelmesh/payload/RemotePayloadProcessor.java

```diff
@@ -64,7 +64,8 @@ private static PayloadContent prepareContentBody(Payload payload) {
         } else {
             data = "";
         }
-        return new PayloadContent(id, modelId, data, kind);
+        String status = payload.getStatus() != null ? payload.getStatus().getCode().toString() : "";
+        return new PayloadContent(id, modelId, data, kind, status);
```

Comment:
It may be good to include the status message as well as the code, but no harm in doing that in a follow-on.